### PR TITLE
Mrc 126 understand financial year

### PR DESCRIPTION
**Pull request by PavelVjalicin**
Implemented

### CODE REVIEW

File: app/forms/DateUtils.scala [hunk 1]
```diff
@@ -22,11 +22,14 @@ import java.time.format.DateTimeFormatter
 object DateUtils {
 
   private val FORMAT = DateTimeFormatter.ofPattern("d MMM yyyy")
+  private val FORMAT_FULL = DateTimeFormatter.ofPattern("d MMMM yyyy")
```
Comment (girishkamat) [inline app/forms/DateUtils.scala:25]: Should we be using consistent date format across all pages? I had spoken to Darryl and he was happy with either MMM or MMMM and we decided to use MMM as it saves space especially on smaller devices

File: app/views/ResultsPageView.scala.html [hunk 2]
```diff
@@ -106,6 +128,15 @@ <h2 class="govuk-heading-m">@messages("resultsPage.yourDetails")</h2>
             </dd>
         </div>
     </dl>
+
+    @calculatorResult match {
```
Comment (girishkamat) [inline app/views/ResultsPageView.scala.html:132]: Wondering if using a fold method (as done in my PR) is a better option as it keeps pattern matches in one place and avoids use of `case _` to cover default cases (fold makes it explicit)

File: app/forms/DateUtils.scala (hunk 1 above)
Comment (PavelVjalicin) [reply app/forms/DateUtils.scala:25]: Sure, I will remove this. Makes the code cleaner.

Comment (PavelVjalicin) [reply app/forms/DateUtils.scala]: Done

File: app/views/ResultsPageView.scala.html (hunk 2 above)
Comment (PavelVjalicin) [reply app/views/ResultsPageView.scala.html]: Done

**Review by girishkamat — APPROVED** [review]

**Pull request merged**